README.rst

```diff
@@ -7,58 +7,103 @@ The structure of the directory looks something like this:
+       |   |   |   |   ├───0.0.1_install.rb
+       |   |   |   |   ├───0.0.1_uninstall.rb
+       |   |   |   |   ├───0.1.2_install.rb
+       |   |   |   |   ├───0.1.2_uninstall.rb
+       |   |   |   |   └───...
+       |   |   |   └───...
+       │   │   ├───recipe_name2
+       |   |   |   └───...
+       │   │   └───...
+       │   ├───<GEri/GEi-name2>
+       │   │   └───...
+       │   └───...
+       ├───puppet-modules
+       |   ├───README.rst
+       |   ├───<GEri/GEi-name1>
+       |   │   ├───module_name1
+       |   |   |   ├───README.rst
+       |   |   |   ├───fiware_release
+       |   │   |   ├───files
+       |   │   |   ├───templates
+       |   │   |   ├───manifests
+       |   │   |   |   ├───init.pp
+       |   │   |   |   ├───install.pp
+       |   │   |   |   ├───uninstall.pp
+       |   │   |   |   └───...
+       |   │   |   └───...
+       |   |   ├───module_name2 
+       |   │   |   └───...
+       |   │   └───...
+       |   ├───<GEri/GEi-name2>
+       |   │   └───...
+       |   └───...
+       |
+   
 
 An overview of what each of these does:
 
 ================  =============
  DIRECTORY         DESCRIPTION
 ================  =============
- chef-recipes      Directory that contains the different chef recipes for all the Generic Enablers.
  images            Directory that contains the script used to create the Generic Enabler images.
+ chef-recipes      Directory that contains the different chef recipes for all the Generic Enablers.
  puppet-modules    Directory that contains the different puppet modules for all the Generic Enablers.
  GEri/GEi-name1    Directory corresponding to each Generic Enabler (reference) instance.
 ================  =============
 
-
+Pay attention that you should include a file named *fiware_release* in your Chef-Recipes and Puppet-Modules where you should
+specify all information about the FIWARE Release of these software.
 
 Last but not least, the repository is structured in branchs:
 
-- master, stable version, at the end of a Release n, we make a pull request from Release n-1 in develop to master.
+- master, stable version, at the end of a Release n, we make a pull request from Release n-1 in develop into master branch
+  and we create a new tag with all stable content of the repo for that FIWARE release.
 - develop, default work branch.
-- feature/<GEri/GEi-name>, branch used to the GEri/GEi owner to upload the content. When you consider that the scripts are correct you should launch a pull request to develop. FIWARE Lab team will check the scripts in order to secure that it is working and if all is correct we will accept the pull request to develop branch.
\ No newline at end of file
+- feature/<GEri/GEi-name>, branch used by the GEri/GEi owner to upload the content.
+
+How to contribute to this repo
+==============================
+
+Probably, you will not have access rights to create branches in this repo. If so, you must request them to FIWARE Lab team 
```

Comment:
Maybe, there is a missunderstanding, I suggest to say something like it:

    "Probably, you will not have access rights to create branches in this repo. We do not give access to other developers except the FIWARE Lab support Team, you should work in your own forked repository and make pull request to our repository."

the rest of description should reflect this idea if we do not want to give access to external developers inside this repo.